process/toc-decision-process.md

```diff
@@ -0,0 +1,70 @@
+# TOC Decision Making Process
+
+This document outlines a decision-making structure used by the TOC
+that takes into account feedback from members of the community
+and strives to find consensus, while avoiding deadlocks.
+
+All non-sensitive discussions take place on GitHub, slack, the TOC
+mailing list and TOC meetings. Sensitive discussions take place
+in private communication channels.
```

Comment:
This is not quite the same as what the [charter](https://github.com/cncf/foundation/blob/main/charter.md) says ("Transparency. The TOC shall hold regular open TOC meetings and all project-related decisions shall be made in those meetings, on a public mailing list, or on public issues.")  In other words, according the the charter the TOC can have private *discussions* about non-sensitive topics. In the past we found this was really helpful because it allowed individual members of the TOC to "think out loud" amongst the smaller group before bringing more fully formed ideas to the larger community.

Reply:
@nikhita Let's point to the language used in the charter to avoid misrepresentation and provide the most flexibility, per Liz's suggestion.